.mailmap

```diff
@@ -629,6 +629,7 @@ Kristian Brünn <hello@kristianbrunn.com> Kristianmitk <hello@kristianbrunn.com>
 Krit Karan <kritkaran.b13@iiits.in> <kritkaran94@users.noreply.github.com>
 Kshitij <kshitijparwani.mat18@itbhu.ac.in> Kshitij Parwani <44468674+P-Kshitij@users.noreply.github.com>
 Kshitij Saraogi <KshitijSaraogi@gmail.com>
+Kuldeep <kuldeepborkar.jr@gmail.com>
 Kuldeep Borkar Jr <kuldeepborkarjr765@gmail.com> Kuldeep Borkar Jr <74557588+KuldeepBorkar@users.noreply.github.com>
```

Comment:
This will generate two separate entries in the AUTHORS file, Are `Kuldeep <kuldeepborkar.jr@gmail.com>` and `Kuldeep Borkar Jr <kuldeepborkarjr765@gmail.com>` the same person or different people?

Reply:
> That makes the second name/email combination an alias of the first.

@oscarbenjamin I see, 
```
name <email> alias
```
Means if I want to add more alias I can just do like this;
```
Kuldeep Borkar Jr <kuldeepborkarjr765@gmail.com> alias
Kuldeep Borkar Jr <kuldeepborkarjr765@gmail.com> alias
```
In the authors file I checked this name and email has been added so I think I should use this as first name/email and make alias in front of that.

Thank you so much for the clarification : )